tests/destructor/tseq_copy_unpack_empty.nim

```diff
@@ -4,6 +4,16 @@ discard """
 showing original type, length, and contents seq[int] 1 @[42]
 copy length and contents 1 @[42]
 '''
+description: '''
+ . From https://github.com/nim-lang/Nim/issues/12037
+   Newruntime: seq copy invalidates original seq
+
+ . From https://github.com/nim-lang/Nim/issues/12820
+   @[] is a problem for --gc:destructors
+
+ . From https://github.com/nim-lang/Nim/issues/12989
+   ARC: Unpacking tuple with seq causes segfault
```

Comment:
As i understand, half of these tests have nothing to do with destructors

Reply:
Sorta, the tricky thing is that semantics of these operations changed between gc, as the ringtones vary.

For unpacking it's not always destructors but there could allocations/copies in which case destruction and or moves.